cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -5690,17 +5744,35 @@ ir.cpp:
 
 # 1015| void OperatorDelete()
 # 1015|   Block 0
-# 1015|     v1015_1(void)              = EnterFunction                    : 
-# 1015|     mu1015_2(unknown)          = AliasedDefinition                : 
-# 1015|     mu1015_3(unknown)          = InitializeNonLocal               : 
-# 1016|     r1016_1(glval<unknown>)    = FunctionAddress[operator delete] : 
-# 1016|     r1016_2(int *)             = Constant[0]                      : 
-# 1016|     v1016_3(void)              = Call[operator delete]            : func:r1016_1, 0:r1016_2
-# 1016|     mu1016_4(unknown)          = ^CallSideEffect                  : ~m?
-# 1017|     r1017_1(glval<unknown>)    = FunctionAddress[operator delete] : 
-# 1017|     r1017_2(String *)          = Constant[0]                      : 
-# 1017|     v1017_3(void)              = Call[operator delete]            : func:r1017_1, 0:r1017_2
-# 1017|     mu1017_4(unknown)          = ^CallSideEffect                  : ~m?
+# 1015|     v1015_1(void)           = EnterFunction                    : 
+# 1015|     mu1015_2(unknown)       = AliasedDefinition                : 
+# 1015|     mu1015_3(unknown)       = InitializeNonLocal               : 
+# 1016|     r1016_1(glval<unknown>) = FunctionAddress[operator delete] : 
+# 1016|     r1016_2(int *)          = Constant[0]                      : 
+# 1016|     v1016_3(void)           = Call[operator delete]            : func:r1016_1, 0:r1016_2
+# 1016|     mu1016_4(unknown)       = ^CallSideEffect                  : ~m?
+# 1017|     r1017_1(glval<unknown>) = FunctionAddress[operator delete] : 
+# 1017|     r1017_2(String *)       = Constant[0]                      : 
+#-----|   Goto -> Block 1
+#-----|   Goto -> Block 3
```

Comment:
This looks wrong. It looks like the call to `delete` _and_ the destructor call is both specified as the successor here.

Reply:
Yeah - there's an underlying extractor issue here that's causing issues with `delete` whenever the object being deleted has a destructor.

Reply:
I see. Do you mind creating an internal issue for this? And is this something we can work around in IR generation for now?